meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/update-resin-supervisor

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
` set -o pipefail` is a bash thing (not POSIX compliant), so we should make that clear (appeases shellcheck).

Reply:
Would you mind adding a `#shellcheck disable` line instead of forcing a bash dependency? Because of having nearly no free space left in the rootfs for some devices it's not unthinkable that we might consider removing bash as a dependency and using busybox sh instead.

Reply:
I think this is categorically the wrong approach, and I'm not really comfortable doing that. If this script relies on non-POSIX compliant features, that should be obvious, especially given that we rely on `pipefail` to catch errors. When making the decision to switch to `sh`, these sorts of `bash` features should be considered and confirmed to behave the same when updated.

Reply:
hi @xginn8 both bash and hush support `pipeline`, I am not speaking about POSIX features but about introducing an artificial dependency on bash when the script as is would run equally well in hush just because a external linter tool complains.
Anyway, if you decide to go ahead with the dependency you should make it explicit and also add it to the RDEPENDS on the recipe.